vacuum.py

```diff
@@ -120,7 +120,9 @@
 }
 
 ALL_PROPS = ["run_state", "mode", "err_state", "battary_life", "box_type", "mop_type", "s_time",
```

Comment:
I think there is a typo (not your fault) : `battery`, not `battary`
```suggestion
ALL_PROPS = ["run_state", "mode", "err_state", "battery_life", "box_type", "mop_type", "s_time",
```